react/features/base/react/components/native/JitsiSectionList.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    SafeAreaView,
+    SectionList
+} from 'react-native';
+
+
+/**
+ * Implements a React/Web {@link Component} for displaying text similar to React
+ * Native's {@code Text} in order to faciliate cross-platform source code.
+ *
+ * @extends Component
+ */
+export default class JitsiSectionList extends Component {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+
+    static propTypes = {
+
+        /**
+         * Rendered when the list is empty. Can be a React Component Class,
+         * a render function, or a rendered element.
+         */
+        ListEmptyComponent: PropTypes.object,
+
+        /**
+         * Optionally define an arbitrary key extractor for this section,
+         * overriding the default keyExtractor.
+         */
+
+        keyExtractor: PropTypes.func,
+
+        /**
+         * If provided, a standard RefreshControl will be added for
+         * "Pull to Refresh"
+         * functionality. Make sure to also set the refreshing prop correctly.
+         */
+
+        onRefresh: PropTypes.func,
+
+        /**
+         * Set this true while waiting for new data from a refresh.
+         */
+
+        refreshing: PropTypes.bool,
+
+        /**
+         * Default renderer for every item in every section.
+         * Can be over-ridden on a per-section basis.
+         * Should return a React element.
+         */
+
+        renderItem: PropTypes.func,
+
+        /**
+         * Rendered at the top of each section.
+         * These stick to the top of the ScrollView by default on iOS.
+         * See stickySectionHeadersEnabled.
+         */
+
+        renderSectionHeader: PropTypes.func,
+
+        /**
+         * The actual data to render
+         */
+
+        sections: PropTypes.array
+    };
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <SafeAreaView>
+                <SectionList
+                    ListEmptyComponent = { this.props.ListEmptyComponent }
+                    keyExtractor = { this.props.keyExtractor }
+                    onRefresh = { this.props.onRefresh }
+                    refreshing = { this.props.refreshing }
+                    renderItem = { this.props.renderItem }
+                    renderSectionHeader = { this.props.renderSectionHeader }
+                    sections = { this.props.sections } />
+
+            </SafeAreaView>
+        );
+
+        // FIXME add styles in SectionList props
```

Comment:
Is this still valid? If yes, please add styles.

Reply:
- [ ] readd styles